src/components/User/index.js

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from "react";
+import { useSelector } from "react-redux";
+import axios from "axios";
+import { METHODS } from "../../services/api";
+import "./styles.scss";
+import { toast } from "react-toastify";
+
+function User() {
+  const user = useSelector(({ User }) => User);
+  const [allClasses, setAllClasses] = useState([]);
+  const [email, setEmail] = useState("");
+
+  useEffect(() => {
+    axios({
+      method: METHODS.GET,
+      url: `${process.env.REACT_APP_MERAKI_URL}/apiDocs/chat/rooms`,
+      headers: {
+        accept: "application/json",
+        Authorization: user.data.token,
+      },
+    }).then((res) => {
+      setAllClasses(res.data);
+    });
+  }, []);
+
+  const changeHandler = (e) => {
+    setEmail(e.target.value);
+  };
+
+  const submitHandler = () => {
+    const notify = () => {
+      toast.success("Added the class successfully", {
```

Comment:
Please change the message to `Added ${user_email} to ${class room name}`

@Poonam-Singh-Bagh